pkg/subctl/operator/lighthouse/ensure.go

```diff
@@ -38,5 +39,11 @@ func Ensure(status *cli.Status, config *rest.Config, operatorNamespace string) (
 		status.QueueSuccessMessage("Created lighthouse service account and role")
 	}
 
+	if created, err := scc.Ensure(config, operatorNamespace); err != nil {
```

Comment:
Ok, this is something that we will need to move into the operator for next cycle, or the OLM definitions, we will see..